services/galley/test/integration/API/Teams/Feature.hs

```diff
@@ -171,6 +172,34 @@ testSearchVisibility = do
     getTeamSearchVisibility tid3 Public.TeamFeatureEnabled
     getTeamSearchVisibilityInternal tid3 Public.TeamFeatureEnabled
 
+testClassifiedDomains :: TestM ()
```

Comment:
The test depends on the config for galley. So the galley config for both environments should be the same to guarantee that the test succeds on both environments:

- for tests on CI: `hack/helm_vars/wire-server/values.yaml` and `/charts/galley/templates/configmap.yaml` need to be adapted
- for development: `services/galley/galley.integration.yaml`

Reply:
Sure, I will align these configuration files.

Should I have this test here that would fetch the classified domains via the endpoint and just assert the values it expects there?